addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -99,7 +101,8 @@ def _compute_nbr_channel(self):
     @api.depends('channel_ids.rating_ids')
     def _compute_percentage_satisfaction(self):
         for record in self:
-            repartition = record.channel_ids.rating_get_grades()
+            dt = (datetime.now() - timedelta(days=7)).strftime('%Y-%m-%d %H-%M-%S')
```

Comment:
Format should be `openerp.tools.DEFAULT_SERVER_DATETIME_FORMAT`. And should probably use `utcnow()`.